### PR TITLE
Aswathy/trade steps

### PR DESCRIPTION
**Pull request by aswathy-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/features/components/organisms/markets/markets-trade-steps/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@use 'features/styles/theme/theme-mixins' as *;
+.item_container {
+    background-color: var(--color-white);
+    box-shadow: rgba(14, 14, 14, 0.1) 0px 4px 8px 0px;
+    inline-size: 31.6rem;
+    padding: 2.4rem;
+    block-size: 16.4rem;
+    position: relative;
+    border-radius: 1.6rem;
```
Comment (NikitK-deriv) [inline src/features/components/organisms/markets/markets-trade-steps/styles.module.scss:9]: these properties and `background-color` can be passed via props in `FlexBox.Box` container instead
And please check are we really need `position: relative;`? 🧐

File: src/features/components/organisms/markets/markets-trade-steps/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+}
+
+.item_description {
+    @include breakpoints(tablet) {
+        min-block-size: 10rem;
```
Comment (NikitK-deriv) [inline src/features/components/organisms/markets/markets-trade-steps/styles.module.scss:14]: Should we use this property for desktop as well?

File: src/features/components/organisms/markets/markets-trade-steps/trade-steps-wrapper.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                lg={{
+                    gap: '12x',
+                }}
+            >
+                {data.map((item) => (
+                    <TradeSteps key={item.id} item={item.data} />
+                ))}
+            </Flex.Box>
+            <Flex.Box container="fluid" justify="center" align="center" mt="20x">
+                <Button.Primary aria-label="Signup" id="dm-nav-signup" onClick={handleSignup}>
```
Comment (NikitK-deriv) [inline src/features/components/organisms/markets/markets-trade-steps/trade-steps-wrapper.tsx:44]:
```suggestion
                <Button.Primary aria-label="Signup" id="dm-nav-signup" onClick={useHandleSignup(is_ppc_redirect)}>
```
Also, are we using default font size for text here?

**Review by mohsen-deriv — CHANGES_REQUESTED** [review]
please check the comments, thanks :pray:

File: src/features/components/organisms/markets/markets-trade-steps/styles.module.scss (hunk 1 above)
Comment (aswathy-deriv) [reply src/features/components/organisms/markets/markets-trade-steps/styles.module.scss]: okay

File: src/features/components/organisms/markets/markets-trade-steps/styles.module.scss (hunk 2 above)
Comment (aswathy-deriv) [reply src/features/components/organisms/markets/markets-trade-steps/styles.module.scss]: okay

File: src/features/components/organisms/markets/markets-trade-steps/trade-steps-wrapper.tsx (hunk 3 above)
Comment (aswathy-deriv) [reply src/features/components/organisms/markets/markets-trade-steps/trade-steps-wrapper.tsx:62]: yes

File: src/features/components/organisms/markets/markets-trade-steps/trade-steps-wrapper.tsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    const { is_ppc_redirect } = usePpc()
+    const handleSignup = useHandleSignup(is_ppc_redirect)
+
+    return (
+        <Container.Fixed
+            as="section"
+            bgcolor="secondary"
+            lg={{ padding: '40x' }}
+            padding_block="20x"
+            padding_inline="8x"
```
Comment (NikitK-deriv) [inline src/features/components/organisms/markets/markets-trade-steps/trade-steps-wrapper.tsx:28]: Just to be cleaner
```suggestion
            padding_block="20x"
            padding_inline="8x"
            lg={{ padding: '40x' }}
```

**Review by NikitK-deriv — APPROVED** [review]
Looks great 🔥 
Could you test it somewhere just to be sure that everything works?

**Review by NikitK-deriv — APPROVED** [review]

**Pull request closed without merging**